src/mysky.ts

```diff
@@ -124,6 +123,13 @@ export class MySky {
   // Public API
   // ==========
 
+  /**
+   * Checks whether the user can be automatically logged in (the seed is present
+   * and required permissions are granted).
+   *
+   * @param perms - The requested permissions.
+   * @returns - Whether the seed is present and a list of granted and rejected permissions.
+   */
   async checkLogin(perms: Permission[]): Promise<[boolean, CheckPermissionsResponse]> {
     log("Entered checkLogin");
```

Comment:
I suppose this logging can be disabled globally?

Reply:
Yes, by passing `debug: false` (the default) in the MySky parameters.